src/test/javascript/spec/helpers/mocks/service/mock-programming-exercise.service.ts

```diff
@@ -26,4 +27,5 @@ export class MockProgrammingExerciseService {
     generateStructureOracle = (exerciseId: number) => of({});
     unlockAllRepositories = (exerciseId: number) => of({});
     getDiffReportForCommits = (exerciseId: number, participationId: number, olderCommitHash: string, newerCommitHash: string, repositoryType: string) => of({});
+    getCheckoutDirectoriesForProgrammingLanguage = (programmingLanguage: ProgrammingLanguage) => of();
```

Comment:
The signature has changed:

```suggestion
    getCheckoutDirectoriesForProgrammingLanguage = (programmingLanguage: ProgrammingLanguage, checkoutSolution: boolean) => of();
```